crates/router/Cargo.toml

```diff
@@ -31,6 +31,7 @@ connector_choice_mca_id = ["api_models/connector_choice_mca_id", "euclid/connect
 external_access_dc = ["dummy_connector"]
 detailed_errors = ["api_models/detailed_errors", "error-stack/serde"]
 payouts = []
```

Comment:
```suggestion
payouts = ["payout_retry"]
```

Reply:
Since, payout_retry is dependent on payout, payout is put under payout_retry